hack/build-podman.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd $(dirname ${BASH_SOURCE[0]})
+
+./podman.sh build
```

Comment:
Do we need to run the podman.sh script from the hack directory for it to work? If so, can we make some changes to be able to run it from the root itself (like other scripts are doing, test-all.sh for example). It seems like we are adding one extra layer of complexity in build-podman.sh for cd'ing to the hack directory.